control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/reconcile.go

```diff
@@ -141,7 +148,11 @@ func ReconcileDefaultIngressPassthroughService(service *corev1.Service, defaultN
 	// secondary networks.
 	service.Spec.Selector = map[string]string{}
 
-	service.Spec.Type = corev1.ServiceTypeClusterIP
+	if hcp.Spec.Platform.Type == hyperv1.KubevirtPlatform {
+		service.Spec.Type = corev1.ServiceTypeClusterIP
+	} else {
+		service.Spec.Type = corev1.ServiceTypeLoadBalancer
```

Comment:
I need to find a way to not having a floating IP created for that LB.
The alternative is to create a Pod (e.g. Nginx) and its `ServiceTypeClusterIP` like Kubevirt did, but for VMs.